Contents/Code/plex_api_helper.py

```diff
@@ -404,9 +408,9 @@ def get_database_info(item):
                     database_id = temp_database_id
                     database = temp_database
 
-                if temp_database == 'themoviedb':
+                if temp_database in ('themoviedb', 'tmdb'):  # tmdb is our prefered db, so we break if found
```

Comment:
```suggestion
                if temp_database == 'themoviedb':  # tmdb is our prefered db, so we break if found
```

`temp_database` cannot be `tmdb` ... it is created from the `guid_map` dictionary

```py
guid_map = dict(
    imdb='imdb',
    tmdb='themoviedb',
    tvdb='thetvdb'
)
```

So this should be reverted... can leave the comment though.

Reply:
Oh. Well. I feel stupid now